openfe/setup/molecule.py

```diff
@@ -20,14 +28,37 @@ class Molecule:
         will be used in the hash.
     """
     def __init__(self, rdkit: RDKitMol, name: str = ""):
-        self._rdkit = rdkit
+        # internally we store as RDKit, for now...
+        self._rdkit = Chem.Mol(rdkit)
         self._hash = hashmol(self._rdkit, name=name)
 
-    # property for immutability; also may allow in-class type conversion
-    @property
-    def rdkit(self) -> RDKitMol:
-        """RDKit representation of this molecule"""
-        return self._rdkit
+    def to_rdkit(self) -> RDKitMol:
+        """Return an RDKit copied representation of this molecule"""
+        return Chem.Mol(self._rdkit)
+
+    @classmethod
+    def from_rdkit(cls, rdkit: RDKitMol, name: str = ""):
+        """Create a Molecule copying the input from an rdkit Mol"""
+        return cls(rdkit=Chem.Mol(rdkit), name=name)
```

Comment:
```suggestion
        return cls(rdkit=rdkit, name=name)
```

since we copy in the `__init__`, no need to double the copying here.